lxd/db/generate/lex/case.go

```diff
@@ -13,6 +13,9 @@ func Capital(s string) string {
 
 // Minuscule turns the first character to lower case ("Foo" -> "foo")
 func Minuscule(s string) string {
+	if s == "UUID" || s == "ID" {
```

Comment:
@masnax rather than adding the special cases here, is it worth adding logic that says "if all chars are uppercase then leave it alone"?

Reply:
Either way the comment on the function needs updating too. Thanks

Reply:
I can't personally think of another all-uppercase field name that might potentially be a database field too, but you're right, that would be safer overall :)

Reply:
I guess "IP" could be another on in the future :)